HelpMyStreetFE/HelpMyStreetFE/Services/Users/UserService.cs

```diff
@@ -163,6 +165,17 @@ private void RefreshUserCache(int userId, CancellationToken cancellationToken)
             }, $"{CACHE_KEY_PREFIX}-user-{userId}", cancellationToken);
         }
 
+        public async Task<IEnumerable<Location>> GetLocations(int userId, CancellationToken cancellationToken)
+        {
+            var user = await GetUserAsync(userId, cancellationToken);
+            var locationResponse = await _addressService.GetLocationsByDistance((int)user.SupportRadiusMiles, user.PostalCode);
```

Comment:
I think we ignore the user's SupportRadius and put in a constant of 20 miles - ideally stored in an app setting.